<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" idbasedtr="true">
 <author>ффф</author>
 <class>MedicALL</class>
 <widget class="QDialog" name="MedicALL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedicALL</string>
  </property>
  <widget class="QPushButton" name="processBTN">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>911</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Назначить лекарственное средство</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="DiagLSH">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>905</width>
     <height>419</height>
    </rect>
   </property>
   <property name="title">
    <string>Симптомы</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>1081</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Возможное лекарство</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="MedOpSH">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="AlchlSH">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>70</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Алкоголем</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Побочные эфекты</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="prdpSH">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>40</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>136</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Совместимость с:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="smkngSH">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>50</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Курением</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="MedDosSH">
     <property name="geometry">
      <rect>
       <x>401</x>
       <y>40</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>401</x>
       <y>21</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Дозировка</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>21</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Лекарство</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="MedSH">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>40</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxLK">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>521</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Личная карточка</string>
    </property>
    <widget class="QCheckBox" name="smcng">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>145</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Курение</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="alchl">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>145</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Употреблчет алкаголь</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>11</y>
       <width>128</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Принимаемые препараты</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>101</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Возраст</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="age">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>130</y>
       <width>37</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PNTfamil">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PNTsname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>74</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PNTname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>52</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delMedBTN">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить препарат</string>
     </property>
    </widget>
    <widget class="QListWidget" name="CurMedSH">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>406</x>
       <y>10</y>
       <width>101</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="editPacientBTN">
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectBTN">
        <property name="text">
         <string>Загрузить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPacientBTN">
        <property name="text">
         <string>Создать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delPacBTN">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>72</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveBTN">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>41</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Диагноз</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Diag">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>70</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>DiagLSH</zorder>
  <zorder>processBTN</zorder>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <tabstops>
  <tabstop>PNTfamil</tabstop>
  <tabstop>PNTname</tabstop>
  <tabstop>PNTsname</tabstop>
  <tabstop>age</tabstop>
  <tabstop>smcng</tabstop>
  <tabstop>alchl</tabstop>
  <tabstop>addPacientBTN</tabstop>
  <tabstop>Diag</tabstop>
  <tabstop>smkngSH</tabstop>
  <tabstop>MedDosSH</tabstop>
  <tabstop>prdpSH</tabstop>
  <tabstop>MedOpSH</tabstop>
  <tabstop>AlchlSH</tabstop>
  <tabstop>processBTN</tabstop>
  <tabstop>MedSH</tabstop>
  <tabstop>saveBTN</tabstop>
  <tabstop>selectBTN</tabstop>
  <tabstop>delMedBTN</tabstop>
  <tabstop>CurMedSH</tabstop>
  <tabstop>editPacientBTN</tabstop>
  <tabstop>delPacBTN</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
